docs/src/examples.md

```diff
@@ -0,0 +1,160 @@
+# Examples
+
+* Create a blade.
+
+  ```julia
```

Comment:
None of the code blocks on this page are rendering for me, should I expect them to without doing anything addition?

Reply:
That's unexpected behavior. Can you confirm that you are still seeing this behavior in the latest push to this branch?

Reply:
It still doesn't work for me. I did some testing and noticed that it is related to how the code is in a bulleted list -- if I take the code outside of the list, then it renders fine. It could be something to do with my environment as it rendered fine when I copied it into another Markdown editor.

Reply:
@wgvozdjak Quick question. Are you directly rendering the Markdown files (e.g., in a browser) or viewing the HTML generated by Julia Documenter package (which takes the Markdown and generates a set of HTML pages)?

If you are directly rendering the Markdown files, then this behavior might be related to the rendering software. I have noticed that different renderers sometimes interpret indentations differently, which leads to differences in rendering.

The `make docs` rule can be used to build the HTML documentation from the Markdown. I tested the Markdown documents to make sure that Documenter rendered them correctly (since Documenter is being used to generated the HTML documentation that is deployed to the GitHub Pages for the project).

Reply:
I'm viewing the HTML generated by Julia Documenter package after running the command listed in the Makefile (`julia --project=docs --compile=min -O0 docs/make.jl`) on my Windows machine.

Reply:
Hmm. That's unexpected. I don't have any good ideas about what the problem could be.

What do you think about (1) testing the docs generated on GitHub when this pull request is merged into "main" and (2) opening a separate ticket to address just the issue of code rendering in bullet lists on your machine?

Reply:
Yes, that makes sense. I created issue #41.